app/components/navbar.hbs

```diff
@@ -53,9 +53,6 @@
         class='nav__element'
         href={{this.STATUS_URL}}
       >Status</a></li>
-    <li>
-      <LinkTo @route={{this.LIVE_URL}} class="nav__element">Live</LinkTo>
```

Comment:
Actually, you can put a flag here too instead of deleting it, so that if the user puts this ?dev=true, the live navigation will be visible to them.

Check for the similar thing in my site below

https://my.realdevsquad.com/
![image](https://github.com/Real-Dev-Squad/website-www/assets/78433013/8eb37223-23d4-4191-92c5-4ffd5c5997b9)

https://my.realdevsquad.com/?dev=true
![image](https://github.com/Real-Dev-Squad/website-www/assets/78433013/61c5a9a9-4845-4bff-bdb1-96769309c290)

Reply:
Hey @rohan09-raj  I have resolved this and also attached the screenshot below please have a look.
Thanks for pointing this out.

![image](https://github.com/Real-Dev-Squad/website-www/assets/85296770/0171b620-0ee8-43bb-87a7-965d1c7719da)
> Screenshot 1

![image](https://github.com/Real-Dev-Squad/website-www/assets/85296770/1e124382-d45b-4044-ba5e-6f81e2982955)
> Screenshot 2

![image](https://github.com/Real-Dev-Squad/website-www/assets/85296770/5370df37-1084-475b-acb2-5cd73defe7ee)
> Screenshot 2